### PR TITLE
X Wartab Route - Fixed some more small errors and added a ton of information (see google doc in description)

### PR DESCRIPTION
**Pull request by WaveWarrior3**
**Please include the route in the name of the PR (for example, `AS Any% - Fix Archie 2`)**

## Route ID (for example, `alpha-sapphire-any-percent`)

x-wartab-route

## Summary of changes

* Fixed some incorrect damage calcs
* Fixed some typos
* Moved post-Ramos shopping to before lucky egg (taking the steps before can increase Hawlucha friendship)
* Added a lot of information from https://docs.google.com/document/d/1H_0mm75B6p59EvPR43HieU1iQCPelsbu4bdfIlACHpc/edit?tab=t.0
* For the merge conflict, I had to add the defense EVs from the Iron to get the Jumpluff damage calc right
* Headbob and I calced out Grant and found it's worth it to try to crit and go for a two turn on Tyrunt with some Atk IVs. Added these situations to the notes for the fight.

## Checklist
* [x] I have tested these changes and Ranger does not display any errors or warnings.
* [x] All image embeds have relative paths.
* [x] The route contains all fights required for the category.

### CODE REVIEW

**Review by Corvimae — APPROVED** [review]

**Comment by Corvimae** [comment]
Please resolve the merge conflicts!

**Comment by WaveWarrior3** [comment]
My bad! I only saw the first merge conflict and also thought that only those with write access could fix it. It's fixed now.

**Comment by WaveWarrior3** [comment]
You might want to hold off on merging, after talking with Headbob there's a few more updates I'm going to make.

**Pull request merged**
